### PR TITLE
feat(balance): tailoring standardization and sanity checks.

### PR DESCRIPTION
**Pull request by VissValdyr**
## Purpose of change
Phone based PRs are pain. I love pain, that's why I am married and making this PR in my holidays on my old galaxy S9.

## Describe the solution
1. Sanity checks for crafting times
2. Groom stuff set to lower levels, but higher auto learn.
3. Faux fur stuff is now also autolearn 
4. Fur stuff now doesn't use survival 1, a lot recipes already didn't, I completed the list
5. Added fur recipes to crafting books.
6. Lowered requirement levels for leather and fur stuff. Because it was a lot higher than most survival equipment.
7. One recipe needed survival 5, made of a lot of stomachs. Ever heard of it? Now it is more sane.

### CODE REVIEW

**Review by scarf005 — COMMENTED** [review]
```diff
- refactor!: tailoring standardization and sanity checks.
+ refactor: tailoring standardization and sanity checks
```
would the changes be major enough to be save incompatible/have breaking changes?

**Comment by VissValdyr** [comment]
> ```diff
> - refactor!: tailoring standardization and sanity checks.
> + feat(balance): tailoring standardization and sanity checks
> ```
> 
> would the changes be major enough to be save incompatible/have breaking changes?

Yeah, not that super huge. I change it

**Comment by RoyalFox2140** [comment]
Most of these crafting time changes are completely insane, 40 minutes to sew together a wool suit? A ton of stuff have their crafting times halved or reduced even further than half. Tailoring isn't supposed to be an incredibly fast process, this is being done with hand tools and is further dis-incentivizing looting correct clothing, combining this with https://github.com/cataclysmbnteam/Cataclysm-BN/pull/3521 and https://github.com/cataclysmbnteam/Cataclysm-BN/pull/4391 it's an overwhelming buff to crafting that was never within intended balance.

**Comment by VissValdyr** [comment]
> Most of these crafting time changes are completely insane, 40 minutes to sew together a wool suit? A ton of stuff have their crafting times halved or reduced even further than half. Tailoring isn't supposed to be an incredibly fast process, this is being done with hand tools and is further dis-incentivizing looting correct clothing, combining this with #3521 and #4391 it's an overwhelming buff to crafting that was never within intended balance.

I keep crafting of tailoring to the same standards of other crafts. Making crafting longer is what DDA did and did wrong. It is not fun. I don't see where the balance is damaged by it. You need to see what other recipes of the same "category" did. A normal duster took 45 minutes. Why should a fur version take so much longer? What is the gameplay reasoning behind it.


Also this lowering of time to craft desensitize crafting some recipes to gain levels.

**Comment by VissValdyr** [comment]
`{
    "result": "duster",
    "type": "recipe",
    "category": "CC_ARMOR",
    "subcategory": "CSC_ARMOR_TORSO",
    "skill_used": "tailor",
    "difficulty": 3,
    "time": "42 m",
    "autolearn": true,
    "book_learn": [ [ "textbook_tailor", 2 ], [ "tailor_portfolio", 2 ] ],
    "using": [ [ "sewing_standard", 32 ] ],
    "components": [ [ [ "rag", 24 ] ] ]
  },`

See here, that regular duster took only 42 minutes. No one bats an eye. So I keep that as the reference.

File: data/json/recipes/armor/suit.json [hunk 1]
```diff
@@ -809,7 +809,7 @@
     "subcategory": "CSC_ARMOR_SUIT",
     "skill_used": "tailor",
     "difficulty": 3,
-    "time": "10 h",
+    "time": "40 m",
```
Comment (VissValdyr) [inline data/json/recipes/armor/suit.json:812]:
```suggestion
    "time": "2 h",
```
I am going to be generous for you @RoyalFox2140

**Comment by RoyalFox2140** [comment]
Comparing proficiencies and theoretical knowledge to BN is a false comparison, and shifting all the way into the opposite direction to making looting meaningless is not the way to go here. Our crafting times were perfectly reasonable, if anything dusters are too fast compared to their fur versions. Your lone example of dusters doesn't factor in the wool suit going from 10 hours of crafting time to 40 minutes, or any of the other examples where things were halved in craft time or more.

Also, to consider with how bad our crafting is right now: Makeshift bandages take seconds to craft but making 10-15 of them will give you level 1 first aid. Shorter recipes still give a great deal of training to your skill level.

A partial audit of craft times is not really warranted here. It has to be a full rework which considers the current state of our balance and factors the effort to tailor custom equipment relative to our spawn chances on clothing to ensure it's not vastly superior to looting. (Given that looting has a fair chance of clothes being flagged as poor fit, and no guarantees to find what you need) Right now you've thrown in a bunch of numbers that don't work with the rest of the state of tailoring or game balance.

**Comment by RoyalFox2140** [comment]
I would like to hear opinions from @Coolthulhu about this because to my understanding from https://github.com/cataclysmbnteam/Cataclysm-BN/pull/1277 this would still fall under our precedent of us wanting to incentivize players to primarily loot what they need, and secondarily doing disassemblies and complex crafting to fill gaps. There was a lot of discussion with this about thread and strings, but I feel this is a greater impact on the game than changes to rag uncrafting.

**Comment by VissValdyr** [comment]
> A partial audit of craft times is not really warranted here. It has to be a full rework which considers the current state of our balance and factors the effort to tailor custom equipment relative to our spawn chances on clothing to ensure it's not vastly superior to looting.


The problem is, who is going to do any audit? Correct, no one. I thing raising the time to craft doesn't do the game a favor. Waiting will change nothing than lowering the fun. For high end stuff looting is still needed, I didn't touch any of the higher class stuff. All is stuff in the early game. So the impact isn't felt by the veteran, with his dead mobile.

The craft times for all those items are still reasonable inside their area. Also as I said, I raised a lot of times, only a few were cut short, especially, when better gear doesn't need as much time to produce!

**Comment by RoyalFox2140** [comment]
> The problem is, who is going to do any audit? Correct, no one. 
The problem is, doing it partially only makes it harder on whoever has to revert your changes if you will only do a couple things. You're changing it piecemeal to what you want it to be, and what you think is fun, without regards to mechanics outside of crafting.

**Comment by VissValdyr** [comment]
> > The problem is, who is going to do any audit? Correct, no one.
> 
> The problem is, doing it partially only makes it harder on whoever has to revert your changes if you will only do a couple things. You're changing it piecemeal to what you want it to be, and what you think is fun, without regards to mechanics outside of crafting.

But I changed it for tailoring. That is not piecemeal. Or did I forget any tailoring recipes?

**Comment by VissValdyr** [comment]
I would also like to put @scarf005 who allowed your 5 minute rag disassemble change into the discussion.

**Comment by scarf005** [comment]
> I would also like to put @scarf005 who allowed your 5 minute rag disassemble change into the discussion.

in this case i'd prefer shorter crafting times (@VissValdyr 's solution). this is because due to performance limitation of 1-second turns, longer crafting times are very painful UX-wise. while balance is subjective, reducing IRL time taken is also important.

i want to stress again that balance is hard; balancing without statistics are even harder. thus we [shouldn't really fight over this](https://github.com/cataclysmbnteam/Cataclysm-BN/blob/main/CODE_OF_CONDUCT.md#our-standards). in this case, merging first then getting player survey for testing whether the idea goes well might be a solution (it's experimental, after all).

**Comment by VissValdyr** [comment]
@scarf005 that reminds me, maybe we should add a second person to the listing if the rules are violated. That is me in there. And since my expressions can get a bit harsh, maybe you or Keenan should be the 2nd in line. You two are actually very kind and down to earth

**Comment by RoyalFox2140** [comment]
> i want to stress again that balance is hard; balancing without statistics is even harder. thus we [shouldn't really fight on this as there's no answer](https://github.com/cataclysmbnteam/Cataclysm-BN/blob/main/CODE_OF_CONDUCT.md#our-standards). in this case, merging first then getting player survey for testing whether the idea goes well might be a solution (it's experimental, after all).

It's hard to have any discussion on balancing without one side feeling like it's unproductive arguing. We have no guidelines on balance or what should be merged, and no sense of direction in the project. Last time we tried making guidelines on balancing the only conclusion we reached was to freeze balance PR's and poll it, but the rest of the discussion on how to balance became arguing on the discord. We need to make guidelines on this.

**Comment by VissValdyr** [comment]
> > i want to stress again that balance is hard; balancing without statistics is even harder. thus we [shouldn't really fight on this as there's no answer](https://github.com/cataclysmbnteam/Cataclysm-BN/blob/main/CODE_OF_CONDUCT.md#our-standards). in this case, merging first then getting player survey for testing whether the idea goes well might be a solution (it's experimental, after all).
> 
> It's hard to have any discussion on balancing without one side feeling like it's unproductive arguing. We have no guidelines on balance or what should be merged, and no sense of direction in the project. Last time we tried making guidelines on balancing the only conclusion we reached was to freeze balance PR's and poll it, but the rest of the discussion on how to balance became arguing on the discord. We need to make guidelines on this.

The problem as usual is, different players want different play styles. Some love long crafting, some love short crafting. DDA has long crafting (even with professions maxed out, see here https://cdda-guide.nornagon.net/item/leg_warmers). BN was about fast gameplay. I am for scarf's approach, by trying, and if a lot are against it, we reverse the problematic parts.

Screenshot_20240428-233647_Opera

**Review by chaosvolt — APPROVED** [review]
This seems overall rather minor to argue over in the first place. Balance-wise, the main thing I'd suggest is consistency. Time taken is sorta a "balancing through tedium" method but also kinda not, in that time can be passed decently quickly unless it's entire days of waiting so the tedium it adds it relatively minor. This is what has allowed a lot of the more time-consuming recipes to stay in BN, as it doesn't get to the point of absurdity like it commonly does in DDA with missing proficiencies.

Everything else looks good, I could go either way regarding the time reductions, I'm approving but leaving it for Scarf or ideally Coolthulhu to leave their thoughts on it though.

**Comment by VissValdyr** [comment]
This is it. I plan no further changes. I tested it and it looks fine. I raised some production timers. (5 minute crafts for some big fur thing)

**Comment by VissValdyr** [comment]
I am asking politely, if a merge can be done soon ish? I usually play with my changes on my phone and I am soon on the way to Singapore, it would be very nice.

**Comment by scarf005** [comment]
> I am asking politely, if a merge can be done soon ish? I usually play with my changes on my phone and I am soon on the way to Singapore, it would be very nice.

please check #4270 regarding applying code changes to local game.

**Comment by VissValdyr** [comment]
> > I am asking politely, if a merge can be done soon ish? I usually play with my changes on my phone and I am soon on the way to Singapore, it would be very nice.
> 
> please check #4270 regarding applying code changes to local game.

Sorry but I can't follow. I can't do it easily with a galaxy phone, because android is blocking the access to the android data folder without USB debugging or jailbreak.

**Comment by chaosvolt** [comment]
Given the disagreements on balance that cropped up in here, I already left my judgment as "would be best to see if Coolthulhu has any objections before merging" so.

**Comment by VissValdyr** [comment]
> Given the disagreements on balance that cropped up in here, I already left my judgment as "would be best to see if Coolthulhu has any objections before merging" so.

Fair.

**Review by chaosvolt — APPROVED** [review]
Welp, we've been waiting a while for a second look at this but...eh, it's good enough I guess?

**Pull request merged**